docs/zh/navigation.yml

```
@@ -4,8 +4,18 @@ nav:
   - 智能应用:
     - 介绍:
       - 什么是智能应用？: dak/intro/index.md
+    - 对话:
+      - 对话页面功能: dak/user-guide/conversation/conversation.md
+    - 语料库:
+      - 创建语料库: dak/user-guide/corpus/create-corpus.md
+      - 上传数据: dak/user-guide/corpus/upload-date.md
+      - 管理语料库: dak/user-guide/corpus/manage-corpus.md
+      - 图文导入: dak/user-guide/corpus/image-upload.md
+      - 格式化导入: dak/user-guide/corpus/format-update.md
     - 插件管理:
       - 创建插件: dak/user-guide/plugins/create-plugin.md
+    - 系统配置:
+      - 系统配置: dak/user-guide/System-Configuration/System-Configuration.md
```

```suggestion
    - 系统配置: dak/user-guide/system-config.md
```
孤行、孤岛太多了。
如果只有一篇文档，可以先不用分目录